crates/sim2h_server/src/bin/sim2h_admin.rs

```diff
@@ -0,0 +1,187 @@
+//! `cargo run --bin sim2h_admin -- --help`
```

Comment:
:thinking: adding more binaries adds more ops works, e.g. we now need to get this building and QA'd cross-platform and see if it has any shared dependencies/libs that need to be on people's machines and might need to add it to dockers and need to roll it into a dist/release process that checks shas in holonix, and need to document how people can interact with it without holonix, etc. (e.g. @samcooley chasing up a live sim2h server box going down recently)...

is there a viable "light touch" approach here, e.g. a bash script in holonix that could suffice for a simple web socket health check?

Reply:
or alternatively, could this be implemented as a `hc` command?

Reply:
yes, lets' move this to hc, that's great.  I've updated the PR title.